setup.py

```diff
@@ -15,10 +17,41 @@ def get_version():
             ver_file.close()
 
 
+def get_data_files():
+
+    def get_completion_install_location(shell):
+        uname = platform.uname()[0]
+        is_root = (os.geteuid() == 0)
+        prefix = ''
+        if shell == 'bash':
+            if is_root and uname == 'Linux':
```

Comment:
if is_root, the bash completion files will be installed at `/etc/bash_completion.d`, but if not, installed at `sys.prefix` + etc/bash_completion.d so it will be like:
* /home/ubuntu/venv/etc/bash_completion.d